src/Nethermind/Nethermind.Era.Test/EraExporterTests.cs

```diff
@@ -32,6 +32,19 @@ public async Task Export_ChainHasDifferentLength_CorrectNumberOfFilesCreated(int
         int fileCount = container.Resolve<IFileSystem>().Directory.GetFiles(tmpDirectory).Length;
         int metaFile = 2;
         Assert.That(fileCount, Is.EqualTo(expectedNumberOfFiles + metaFile));
+        string[] files = Directory.GetFiles(tmpDirectory);
+        foreach (string file in files)
+        {
+            if (Path.GetFileName(file).Equals("accumulators.txt") || Path.GetFileName(file).Equals("checksums.txt"))
+            {
+                foreach (string line in File.ReadLines(file))
+                {
+                    string[] words = line.Split(' ');
+                    Assert.That(words.Length, Is.EqualTo(2));
+                    Assert.That(words[1].EndsWith(".era1"));
```

Comment:
It is tests so minor, but `line.AsSpan().Split(' ');` and then asserting on spans will be slightly less allocatey